.gitignore

```diff
@@ -0,0 +1 @@
+.DS_Store
```

Comment:
I think you weren't supposed to include this file.

Reply:
You're right, I must have messed that up while I was merging